### ISSUE TITLE
Дневная Кумулятивная Дельта

### ISSUE THREAD

**Issue by Vadim174Chel** [SBProX/SBProX-Dev#1022]
Сейчас Кумулятивная Дельта, на платформе SBProX, отображается в виде столбиков, гистограммы, на данный момент не совсем удобна в использовании. Добавьте пожалуйста отображение Кумулятивной Дельты хотя бы в виде линии, если нет возможности сделать отображение в виде свечей. Заранее огромное спасибо.

**Comment by Trim83** [comment]
Полностью поддерживаю данное предложение.

**Comment by Centr27** [comment]
Поддерживаю

**Comment by QuasarCpt** [comment]
Поддерживаю, было бы очень удобно!)

**Comment by gios76** [comment]
согласен... очень хорошая идея отобразить кумулятивную дельту и дневную кумулятивную дельту в виде линии. так лучше будет видно дивергенция кумулятивной дельты и цены, что очень помогает торговле.

**Comment by Vadim174Chel** [comment]
Уважаемые разработчики, снова обращаюсь к вам с огромной просьбой. Сделайте пожалуйста отображение индикатора, Кумулятивная дельта,  в виде японских свечей, как это сделано на других платформах, или хотя бы в виде линии. С индикатором, Кумулятивная дельта, в виде гистограммы совершенно не возможно работать!!!
